static/js/controllers/reports-add.js

```diff
@@ -17,10 +17,19 @@ angular.module('inboxControllers').controller('ReportsAddCtrl',
 
     var getSelected = function() {
       if ($state.params.formId) { // adding
-        return $q.resolve({ form: $state.params.formId });
+        return $q.resolve({
+          formInternalId: $state.params.formId
```

Comment:
It refers to the `internalId` field on the form doc. Yes `P` for pregnancy form. It's internal because it's not the UUID I guess?